### ISSUE TITLE
A11y_ML .NET Model Builder_ Consume_ Keyboard: Keyboard users will not be able to see the complete code snippet in the Consume scenarios

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1573]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #WCAG1.3.1; #Keyboard; #A11yPreview;

### Environment Details:
Microsoft Visual Studio

### Repro Steps:

1. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#)
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Text Classification" and press enter
5. Select "Local (CPU)" environment and tab till "Next" button and press enter
6. In "Data Source Type" and select "File" radio button
7. Tab till "Browse" button and select the input file
8. Tab till "Column to predict" control and select a value and then tab till "Next" button and press enter
9. Tab till "Start Training" button and press enter
10. Once training is completed tab till "Next" button and press enter
11. In "Evaluate" tab navigate till "Next" button and press enter
12. In consume tab navigate till code snippet and verify whether keyboard user will be able to see the complete code snippet

### Actual Result:
Keyboard users will not be able to see the complete code snippet in the Consume scenarios

### Expected Result:
Keyboard users should be able to see the complete code snippet

### User Impact:
Keyboard users may miss the code if complete code snippet is not available for keyboard users

**Comment by NiharikaTipirishetty** [comment]
Keyboard users will not be able to see the complete code snippet in the Consume step in 'Image Classification' and 'Object detection' scenarios as well

**Comment by beccamc** [comment]
Validate on rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. Code snippet in Consume tab is now keyboard accessible.

https://user-images.githubusercontent.com/83570638/131454199-a0c2518d-5b2a-4676-979f-9daffa6bfb1d.mp4

**Comment by akashelijah05** [comment]
#Closed;
